.github/ISSUE_TEMPLATE/pre-release-getting-started-scripts.md

```diff
@@ -0,0 +1,66 @@
+---
+name: Pre-Release Getting Started Script Update
+about: This template can be used to track the update of the "Getting Started" scripts in this repository leading up to the next Stackable release
+title: "chore(tracking): Check and update getting-started scripts"
+labels: ['epic']
+assignees: ''
+---
+
+<!--
+    Make sure to update the link in '.github/ISSUE_TEMPLATE/release.md' when
+    you change the front matter above.
+-->
+
+<!--
+    DO NOT REMOVE THIS COMMENT. It is intended for people who might copy/paste from the previous release issue.
+    This was created by an issue template: https://github.com/stackabletech/issues/issues/new/choose.
+-->
+
+## Pre-Release Getting Started Script Updates
+
+Part of <https://github.com/stackabletech/issues/issues/TRACKING_ISSUE>
+
+Replace the items in the task lists below with the applicable Pull Requests
```

Comment:
There are often no changes to be made, so PRs would only need to be made in exceptional circumstances.

```suggestion
Replace the items in the task lists below with the applicable Pull Requests (if any).
```